jobs/terraform-deploy/terraform-deploy.groovy

```diff
@@ -46,8 +46,7 @@ terraform ${action} -var env=${environment_name} -input=false ${extraopts}
 # AWS
 cd ../aws
 . ~/.aws_credentials
-terraform ${action} -var env=${environment_name} -input=false ${extraopts} \
--var "aws_access_key=${aws_access_key}" -var "aws_secret_key=${aws_secret_key}"
+terraform ${action} -var env=${environment_name} -input=false ${extraopts}
```

Comment:
It feels to me like the following might be easier to understand, wouldn't result in duplicate variables in the vault, and would protect us if the variables were renamed in alphagov/tsuru-terraform (I wouldn't think to check here):
```
-var "aws_access_key=${AWS_SECRET_ACCESS_KEY}" -var "aws_secret_key=${AWS_SECRET_ACCESS_KEY}"
```
What do you think?